### PR TITLE
Enzyme setup and DateHeader tests

### PR DESCRIPTION
**Pull request by jordanstholmes**
@FEC-Kickstand/kickstand if anybody wants to take a look at this, I'd appreciate it. This is the last branch that's a mess, so after this, I shouldn't have anymore disorganized pull requests.
This PR is basically for setting up circle ci and enzyme, with a few tests added in.

Please ignore the db files, including the seeding files. Somebody already looked at those. But they weren't in this branch, so they're showing up in the pull request.

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './app.css';
+
```
Comment (KStockley) [inline client/components/app.jsx]: Why the newline here?

File: client/components/dateHeader.test.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  test('Spacer defaults to left side', () => {
+    expect(
+      wrapper
+        .children()
+        .first()
+        .hasClass('date')
+    ).toBe(true);
+  });
+});
```
Comment (KStockley) [inline client/components/dateHeader.test.jsx]: Decent testing rigour; I like it. Did the linter complain about an end of file newline?

File: client/components/app.jsx (hunk 1 above)
Comment (jordanstholmes) [reply client/components/app.jsx]: Good catch :)

File: client/components/dateHeader.test.jsx (hunk 2 above)
Comment (jordanstholmes) [reply client/components/dateHeader.test.jsx]: Yes

**Pull request merged**